File: 20170716-rust/fib/python/fib.py
#!/usr/bin/env python3

def fib(n: int) -> int:
    if n < 3:
        return 1
    else:
        return fib(n - 1) + fib(n - 2)

if __name__ == '__main__':
    print("{}".format(fib(40)))

File: 20170716-rust/fib/c/fib.c
#include <stdio.h>

int fib(int n) {
    if (n < 3) {
        return 1;
    } else {
        return fib(n - 1) + fib(n - 2);
    }
}

int main() {
    printf("%d\n", fib(40));
    return 0;
}
